server/app.js

```diff
@@ -112,7 +112,7 @@ module.exports = function createApp({ signInService, incidentService, offenderSe
         debug: true,
         outputStyle: 'compressed',
         prefix: '/stylesheets/',
-        includePaths: ['node_modules/govuk-frontend'],
+        includePaths: ['node_modules/govuk-frontend/govuk'],
```

Comment:
not sure about this.  They've gone out of their way to ensure that they prefix everything with `govuk` to ensure that it doesn't conflict and we're then ignoring it.  Their suggested import is
```@import "govuk/all";
``` 
so that `govuk` still there

Reply:
The same applies with `all.js` that we are merging into one assets directory first.

Reply:
agreed and updated